nibe/console_scripts/convert_csv.py

```diff
@@ -16,7 +16,10 @@
 def update_dict(d: MutableMapping, u: Mapping, removeExplicitNulls: bool) -> Mapping:
     for k, v in u.items():
         if v is None and removeExplicitNulls:
-            d.pop(k, None)
+            try:
+                d.pop(k)
+            except (IndexError, KeyError):
+                pass
```

Comment:
Turns out the d object is a pandas object here and not a dict. pop there does not take a default argument.  I think we don't recursively flatten the pandas object to a nested dict.